.pipelines/templates/run-upstream-e2e.yaml

```diff
@@ -0,0 +1,12 @@
+steps: 
+  - bash: |
+      git clone -b ${UPSTREAM_BRANCH} $(upstream.repo) $(repo.path)/osm
+    displayName: Checkout $(upstream.repo) @ ${UPSTREAM_BRANCH}
+  - bash: |
+      cd osm
+      make build-osm 
+      export CTR_REGISTRY=${UPSTREAM_DOCKER_REGISTRY}
+      export CTR_TAG=${UPSTREAM_BRANCH}
+      go test ./tests/e2e -test.v -ginkgo.v -ginkgo.progress -test.timeout 0 -installType=NoInstall -OsmNamespace=arc-osm-system
```

Comment:
we should set a timeout here

Reply:
These upstream e2es usually take around 20 mins. Would 30 mins be a safe timeout?

Reply:
On second thought, I think 60 would be safer, considering we also use this for the timeout for the other e2es.